packages/dev/core/src/FlowGraph/flowGraphEventBlock.ts

```diff
@@ -0,0 +1,24 @@
+import { FlowGraphWithOnDoneExecutionBlock } from "./flowGraphWithOnDoneExecutionBlock";
+
+/**
+ * @experimental
+ * A type of block that listens to an event observable and activates
+ * its output signal ("onTriggered"), when the event is triggered.
+ */
+export abstract class FlowGraphEventBlock extends FlowGraphWithOnDoneExecutionBlock {
+    /**
+     * @internal
+     */
+    public abstract _start(): void;

```

Comment:
I find `start` and `stop` a little too generic... When I read "start", I wonder "start what?".

What about `observe` / `unobserve`? Or `startObserving` / `stopObserving`?

Maybe @bghgary will have some good names for you :)